downstreamadapter/eventcollector/event_collector.go

```diff
@@ -278,8 +297,10 @@ func (c *EventCollector) processLogCoordinatorRequest(ctx context.Context) {
 }
 
 // mustSendDispatcherRequest will keep retrying to send the dispatcher request to EventService until it succeed.
-// Caller should avoid to use this method if the remote EventService maybe offline forever.
+// Caller should avoid to use this method when the remote EventService is offline forever.
 // And this method may be deprecated in the future.
+// FIXME: Add a checking mechanism to avoid sending request to offline EventService.
```

Comment:
Shall we also create a issue for this? to prevent too much FIXME and TODO in the code base ?

Reply:
Good idea.